openassessment/templates/openassessmentblock/grade/oa_grade_complete.html

```diff
@@ -249,89 +106,92 @@ <h3 class="submission__feedback__title ui-toggle-visibility__control">
+                                                           class="option__input feedback__overall__value"
+                                                           value="These assessments were not useful."
+                                                           aria-labelledby="feedback__overall__prompt"/>
+                                                    <label class="option__label" for="feedback__overall__value--notuseful">{% trans "These assessments were not useful." %}</label>
+                                                </li>
+                                                <li class="option option--disagree">
+                                                    <input type="checkbox"
+                                                           name="feedback__overall__value"
+                                                           id="feedback__overall__value--disagree"
+                                                           class="option__input feedback__overall__value"
+                                                           value="I disagree with one or more of the peer assessments of my response."
+                                                           aria-labelledby="feedback__overall__prompt"/>
+                                                    <label class="option__label" for="feedback__overall__value--disagree">{% trans "I disagree with one or more of the peer assessments of my response." %}</label>
+                                                </li>
+                                                <li class="option option--inappropriate">
+                                                    <input type="checkbox"
+                                                           name="feedback__overall__value"
+                                                           id="feedback__overall__value--inappropriate"
+                                                           class="option__input feedback__overall__value"
+                                                           value="Some comments I received were inappropriate."
+                                                           aria-labelledby="feedback__overall__prompt"/>
+                                                    <label class="option__label" for="feedback__overall__value--inappropriate">{% trans "Some comments I received were inappropriate." %}</label>
+                                                </li>
+                                            </ol>
                                         </li>
-                                    </ul>
+                                        <li class="field field--textarea feedback__remarks" id="feedback__remarks">
+                                            <label for="feedback__remarks__value">{% trans "Please provide any feedback on the grade or comments that you received from your peers." %}</label>
```

Comment:
@andy-armstrong Please strike "Please" and "any":
"Provide feedback on the grade or comments that you received from your peers."